module2/homework/nwdNww/nwdNww.hpp

```diff
@@ -1,11 +1,13 @@
 #pragma once
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+   if (rhs!=0)
+        return NWD (rhs,lhs%rhs);
+        return std::abs(lhs); 
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    if (lhs==0 || rhs==0)
+        return 0;
+        return std::abs(lhs * rhs)/NWD(lhs,rhs);
```

Comment:
Formatting: Here for example you have `lhs * rhs` so it's nice - with spaces. And later you have `oneThing/anotherThing` - would be nice to add space to get `oneThing / anotherThing`.